swap_meet/vendor.py

```diff
@@ -1,2 +1,76 @@
+
 class Vendor:
-    pass
+    def __init__(self, inventory=None):
+        if inventory:
+            self.inventory = inventory
+        else:
+            self.inventory = []
+            
+
+    def add(self, add_new_item):
+        self.inventory.append(add_new_item)
+        return (add_new_item)
+
+    def remove(self, remove_item):
+        if remove_item in self.inventory:
+            self.inventory.remove(remove_item)
+            return (remove_item)
+        else:
+            return False
```

Comment:
*Minor note*:   Since the prior `if` block ends in a return, you can leave off the `else` here.

```suggestion
        return False
```